src/modules/rlm_couchbase/configure.ac

```diff
@@ -96,6 +96,38 @@ if test x$with_[]modname != xno; then
 		fail="$fail libcouchbase"
 	fi
 
+	if test "x$have_couchbase" = "xyes"; then
+	  dnl #
+	  dnl #  Now check that the header versions match the library
+	  dnl #
+	  AC_MSG_CHECKING([libcouchbase 3.3.x library and header version consistency])
+	  AC_RUN_IFELSE(
+	    [AC_LANG_PROGRAM(
+	      [[
+	        #include <libcouchbase/couchbase.h>
+	      ]],
+	      [[
+	        printf("library: %#lx ", (unsigned long) LCB_VERSION);
+	        if (LCB_VERSION >= 0x030300) {
+	          return 0;
+	        } else {
+	          return 1;
+	        }
+	      ]]
+	    )],
+	    [
+	      AC_MSG_RESULT(yes)
+	    ],
+	    [
+	      AC_MSG_RESULT(no)
+	      AC_MSG_FAILURE([libcouchbase 3.3.x library version does not match header version])
+	    ],
+	    [
+	      AC_MSG_RESULT([cross-compiling (assuming yes)])
```

Comment:
drop the `AC_MSG_RESULT([cross-compiling (assuming yes)])` and reuse similar steps like in the src/lib/ldap

Reply:
You need to add text to fail i.e. `fail=$fail libcouchbase > 3.3.x`